lib/registries/addon/components/registration-form-view/styles.scss

```
@@ -19,3 +19,105 @@
 .Answer > p {
     font-size: 16px;
 }
+
+.NavigationMenu {
+    position: absolute;
+    z-index: 100;
+    top: 22px;
+    right: 30px;
+    width: 200px;
+    box-shadow: 0 1px 2px $primary-box-shadow;
+}
+
+.Header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    border-bottom: 1px solid $alto;
+}
+
+.HeaderText {
+    padding-left: 15px;
+    flex-basis: 80%;
```

This value depends on the width of the header. If it gets wider, there's a gap between the header text and the "x" icon. Instead of playing with interdependent values until they look right, try to use CSS that works no matter the situation. Flexbox is great.
```suggestion
    flex-grow: 1;
```